### PR TITLE
Package dependencies update and goerli to legacy

### PR DESCRIPTION
**Pull request by kupermind**
- Package dependencies update to get rid of errors when yarn installing;
- `goerli` mentioning to legacy;
- NO CODE CHANGE.

### CODE REVIEW

File: .github/workflows/workflow.yaml [hunk 1]
```diff
@@ -15,10 +15,8 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Get a specific Node.js version such that hardhat does not hang periodically
```
Comment (kupermind) [inline .github/workflows/workflow.yaml:18]: This no longer works, need to get v18.20, and the bug seems to be fixed a long time ago.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "autonolas-tokenomics",
-  "version": "1.0.0",
```
Comment (kupermind) [inline package.json:3]: Adjusting to the Tokenomics contract version

File: package.json [hunk 3]
```diff
@@ -56,12 +57,8 @@
     "@uniswap/v2-core": "^1.0.1",
     "@uniswap/v2-periphery": "^1.1.0-beta.0",
     "canonical-weth": "^1.4.0",
-    "cross-fetch": "^4.0.0",
     "eth-permit": "^0.2.3",
     "ethereum-sources-downloader": "^0.1.21",
-    "node-fetch": "^3.3.2",
-    "scribble-generator": "^1.1.0",
```
Comment (kupermind) [inline package.json:63]: These are not explicitly needed, scribble needs to be installed as a global env anyway if anybody wants to run it.

File: scripts/deployment/README.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ These scripts correspond to the steps in the full deployment procedure (as descr
 For the Tokenomics 1.2, refer to the following deployment procedure: [deployment_v1.2.md](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/deployment_v1.2.md).
 
 ## Observations
-- There are several files with global parameters based on the corresponding network. In order to work with the configuration, please copy `gobals_network.json` file to file the `gobals.json` one, where `network` is the corresponding network. For example: `cp gobals_goerli.json gobals.json`.
```
Comment (kupermind) [inline scripts/deployment/README.md:8]: The rest is just putting `goerli` mentioning in text to legacy

**Review by 77ph — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**